packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/TileProviderController.java

```diff
@@ -84,9 +84,12 @@ public void success(Object data) {
     public void error(String errorCode, String errorMessage, Object data) {
       Log.e(
```

Comment:
I want to call out that this log message is incorrect It uses errorCode for code and message. That said I am not modifying that logic in this PR only reformatting the string format b/c the linter had an issue.